src/django/api/models.py

```diff
@@ -2049,6 +2049,35 @@ class Meta:
     updated_at = models.DateTimeField(auto_now=True)
 
     def __str__(self):
-        return ('EmbedField {id} - EmbedConfig {embed_config}, '
-                'Order: {order} '
-                ).format(**self.__dict__)
+        return (
+            'Column Name {column_name} - Order: {order} ').format(
+            **self.__dict__)
+
+
+class NonstandardField(models.Model):
+    """
+    Nonstandard data fields available to include on facilities in an
+    embedded map
+    """
```

Comment:
Are non-standard fields different than contributor/custom fields?

Reply:
Yes/No. Previously, we were grabbing nonstandard/extra fields that a contributor sent with facilities and, on the fly, getting a collection of them and returning them for use on the settings page, but that was too slow. Now we are creating an NonstandardField object whenever a contributor submits a facility, so that we can just query the NonstandardField objects for that contributor when we go to the settings page. 
Then, on the settings page, we allow the contributor to select which items from their collection of NonstandardFields they want to include on their facilities (creating EmbedField objects), and optionally choose alternative display names for them. 
Finally, we calculate and return the selected display names and the facility-specific values to the facility details API for presentation to users ('contributor_fields' in the facility details request). 
So I'm thinking of it as, we have nonstandard fields that are submitted, then we create embed configuration fields, and then we send the actual facility's contributor-submitted fields data... but it's possible I'm making a false distinction and/or other naming conventions might be clearer.